gramps/gen/plug/_pluginreg.py

```diff
@@ -1123,7 +1128,7 @@ def scan_dir(self, dir, uistate=None):
         extlen = -len(ext)
         pymod = re.compile(r"^(.*)\.py$")
 
-        for filename in os.listdir(dir):
+        for filename in filenames:
             name = os.path.split(filename)[1]
```

Comment:
Yes, but I mean that every value in `filenames` is a single file in `dirpath`. There is no subdirectory, so this `split` does nothing.